### PR TITLE
CAS-463 MessageInputView scrollbar customization

### PR DESCRIPTION
**Pull request by bychkovdmitry**
https://stream-io.atlassian.net/browse/CAS-463

### Description

Added an ability to customize scrollbar for `MessageInputView`:
- enable/disable scrollbar
- possibility to turn on/off fading behavior
- possibility to change thumb width
- possibility to customize thumb drawable

Notes: 
Had to enable scrollbars in corresponding layouts by default, otherwise it is impossible to show them programmatically. Disabling the scrollbar programmatically works as expected.

### Checklist

- [X] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [X] PR targets the `develop` branch
- [X] Changelog updated with client-facing changes
- [ ] New code is covered by unit tests
- [X] Comparison screenshots added for visual changes
- [X] Reviewers added

### CODE REVIEW

File: UNRELEASED_CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 ## stream-chat-android
 - Improve Channel List Diff
+Scrollbar customization for `MessageInputView` via `streamInputScrollbarEnabled`, `streamInputScrollbarFadingEnabled`, `streamInputScrollbarWidth` and `streamInputScrollbarThumb` attributes
```
Comment (zsmb13) [inline UNRELEASED_CHANGELOG.md:5]: Not a list item 🙈

File: stream-chat-android/src/main/java/com/getstream/sdk/chat/style/ScrollbarStyle.kt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            inputScrollbarThumb?.let {
+                textView.verticalScrollbarThumbDrawable = it
+            }
+        }
+    }
+
+    public class Builder(private val array: TypedArray) {
+        private val style: ScrollbarStyle = ScrollbarStyle()
+
+        public fun scrollbarEnabled(@StyleableRes res: Int, defValue: Boolean = false): Builder = apply {
```
Comment (zsmb13) [inline stream-chat-android/src/main/java/com/getstream/sdk/chat/style/ScrollbarStyle.kt:40]: Do these `defValue` parameters need default values?

File: stream-chat-android/src/main/java/com/getstream/sdk/chat/style/ScrollbarStyle.kt [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+        if (inputScrollbarEnabled) {
+            textView.isScrollbarFadingEnabled = inputScrollbarFadingEnabled
+
+            if (inputScrollbarWidth != UNSET_SCROLLBAR_WIDTH) {
+                textView.scrollBarSize = inputScrollbarWidth
+            }
+
+            inputScrollbarThumb?.let {
+                textView.verticalScrollbarThumbDrawable = it
```
Comment (zsmb13) [inline stream-chat-android/src/main/java/com/getstream/sdk/chat/style/ScrollbarStyle.kt:32]: This call seems to require API 29

https://developer.android.com/reference/android/view/View#setVerticalScrollbarThumbDrawable(android.graphics.drawable.Drawable)

**Review by zsmb13 — CHANGES_REQUESTED** [review]
Implementation looks all good, but it looks to me like we won't be able to set the drawable below API 29 (unless we use reflection, apparently)

**Comment by bychkovdmitry** [comment]
The only customizations which work starting from min SDK 21 are: enable/disable scroll bar, enable/disable scroll bar fading behavior. Unfortunately, there is no way to customize the look of scroll bar.

**Review by zsmb13 — APPROVED** [review]

**Pull request merged**